linux_os/guide/services/docker/docker_selinux_enabled/rule.yml

```diff
@@ -21,8 +21,6 @@ rationale: |-
 
 severity: high
 
-platform: machine  # The check uses service_... extended definition, which doesnt support offline mode
```

Comment:
Limitation of OpenSCAP. Content needs to work across all SCAP scanners accurately. Reverting this change.

Reply:
ditto

Reply:
@shawndwells AFAIK, SELinux is not namespaced so containers cannot have their own separate SELinux policies by default. SELinux also appears to be disabled in a container, although it is running on the host operating system. So you cannot really have this rule in a container (when we consider the default container configuration).